ir_attachment_conversion_queue_job/models/ir_attachment.py

```diff
@@ -11,34 +11,13 @@
 class Irattachemnt(models.Model):
     _inherit = "ir.attachment"
 
-    def _product_job_queue(self):
-        attachments = self.search(
-            [
-                ("res_model", "=", "product.template"),
-                ("res_field", "=", "image"),
-                ("res_id", "!=", False),
-            ]
-        )
-        for attachment in attachments:
-            self.with_delay().convert_image_attachments(attachment)
-
-    def convert_image_attachments(self, attachment):
-        try:
-            self.with_delay().convert_image_attachments()
-            self.env["product.template"].browse(
-                attachment.res_id
-            ).image_1920 = attachment.datas
-        except Exception as e:
-            _logger.error(
-                "Error while recovering for %s: %s",
-                attachment.res_id,
-                repr(e),
-            )
-
-    def _product_image_job_queue(self):
+    def _image_job_queue(self):
         mapping = {
+            "product.template": "image",
             "product.image": "image",
             "product.public.category": "image",
+            "res.partner": "image",
+            "res.users": "image",
         }
         for model, field in mapping.items():
             attachments = self.search(
```

Comment:
Shouldn't the search condition be adjusted to exclude already processed ones?

Reply:
I was thinking if we start another migrations (sst15test3, production), there will not be processed ones. That's why I didn't think about excluding.